app/src/main/java/com/ballboycorp/com/topik/SonarBugSample.kt

```diff
@@ -0,0 +1,19 @@
+package com.ballboycorp.com.topik
+
+interface Sample {
+    fun method()
+
+    fun anotherMethod()
+}
+
+class SampleImpl : Sample {
+    override fun method() {
```

Comment:
Add a nested comment explaining why this function is empty or complete the implementation. [Details](https://fb41-221-141-140-219.jp.ngrok.io/project/issues?id=musooff_Topik_AYB-ZFlS8-xhCYxQvxYf&issues=AYB-q2G40n16_HoDUh6x&open=AYB-q2G40n16_HoDUh6x)